src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/OrderReturnStates/create.html.twig

```diff
@@ -29,6 +29,7 @@
 {% extends 'PrestaShopBundle:Admin:layout.html.twig' %}
 
 {% block content %}
+  {% if displayAllShopsContextForced %}{{ ps.infotip(multistoreForcedContextInfoTip, true) }}{% endif %}
```

Comment:
It should be a dedicated template that you can be included. This template will handle the condition :)

Reply:
You will be able to put it in a dedicated block, so people will be able to easily customize it